### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/calyxos/seedvault/) for CalyxOS/Seedvault.

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by chirayudesai** [comment]
Let's get this merged. Weblate usually pulls in changes by itself, but if not we can do a manual rebase before merge.

I believe @mikeNG fixed any issues with the translations weblate side.

**Review by stevesoltys — APPROVED** [review]

File: app/src/main/res/values-es-rUS/strings.xml [hunk 1]
```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about_source_code">Código fuente: https://github.com/stevesoltys/seedvault</string>
+    <string name="about_sponsor">Patrocinado por: &lt;a href=\"https://www.calyxinstitute.org\"&gt;Calyx Institute&lt;/a&gt; for use in &lt;a href=\"https://calyxos.org\"&gt;CalyxOS&lt;/a&gt;<a href="https://www.calyxinstitute.org">Calyx Institute</a> for use in <a href="https://calyxos.org">CalyxOS</a></string>
```
Comment (chirayudesai) [inline app/src/main/res/values-es-rUS/strings.xml:4]: oh wait, these should all be <>

**Comment by chirayudesai** [comment]
Woops, I missed that. Will have to fix via weblate.

**Comment by chirayudesai** [comment]
I just ended up fixing the strings via good old `sed`.

https://github.com/stevesoltys/seedvault/pull/134

Will close this once that is merged.

**Comment by chirayudesai** [comment]
Merged #134 instead.

**Pull request closed without merging**